scripts/userbenchmark/upload_s3.py

```diff
@@ -34,7 +33,7 @@ def upload_s3(ub_name: str, platform_name: str, date_str: str, file_path: Path):
     parser.add_argument("--userbenchmark_platform", required=True,
                         help='Name of the userbenchmark platform')
     parser.add_argument("--userbenchmark_json", required=True,
-                        help='Upload userbenchmark json data')
+                        help='Upload userbenchmark json or regression yaml file.')
```

Comment:
```suggestion
                        help='Upload userbenchmark json and regression yaml file.')
```
I forget what this flag does now...is it just a True/False whether we upload stats?

Reply:
It specifies the file we want to upload to S3. Previously it should be just metrics-*.json file, now we want it to upload regression-*.yaml files as well.

Reply:
Hmmmm would this be clearer as just "--file-to-upload" or "--files-to-upload" then?

Reply:
Right, let me change it to "--upload-file".